protocol/directory.go

```diff
@@ -31,15 +36,18 @@ func InitDirectory(epDeadline merkletree.TimeStamp, vrfKey vrf.PrivateKey,
 	return d
 }
 
-func (d *ConiksDirectory) Update() {
+func (d *ConiksDirectory) Update(up *UpdatePolicies) {
```

Comment:
> If it's just a time interval, clients have to compute for themselves if the server met the deadline

Is that what we do to avoid the clock synchronization issues? I remember that we agreed to let client compute the timestamp for themselves (on hangout). If `EpochDeadline` is an actual time stamp, I think there has not much advantages when using `Epoch` as counter ?